src/earn/poolInfoScreen/Cards.tsx

```diff
@@ -304,23 +311,29 @@ export function AgeCard({
   return (
     <View style={styles.card} testID="AgeCard">
       <View style={styles.cardLineContainer}>
-        <View style={styles.cardLineLabel}>
+        <View style={styles.flexLabel}>
           <LabelWithInfo
             onPress={onInfoIconPress}
             label={t('earnFlow.poolInfoScreen.ageOfPool')}
-            labelStyle={styles.cardTitleText}
+            labelStyle={styles.lineLabel}
             testID="AgeInfoIcon"
+            numberOfLines={undefined}
           />
         </View>
-        <Text style={styles.cardTitleText}>{formattedDuration(dateInterval)}</Text>
+        <View style={styles.flexValue}>
+          <Text style={styles.lineValue}>{formattedDuration(dateInterval)}</Text>
+        </View>
       </View>
     </View>
   )
 }
 
 export const styles = StyleSheet.create({
-  flexShrink: {
-    flexShrink: 1,
+  flexLabel: {
+    flex: 1.2,
```

Comment:
Why the non-standard flex value here? Is this to make up for the lack of the `paddingRight` that we used to have?